python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/tests/example.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
```

Comment:
It'll be nice to organize lambda fixtures in dedicated folder (so they're not confused with other test logic).
e.g. in Node.js we have `fixtures/lambdas` -> https://github.com/serverless/console/tree/c1d85ac7185919f63bbb0d2de40864208e1e80bb/node/packages/aws-lambda-sdk/test/fixtures/lambdas

Reply:
Coming up next: I'll reorganize fixtures in dedicated folder, as well as create a script for building the lambda layer so that we can reuse it in tests 👍

Reply:
I've pushed the changes related to test refactoring (fixtures) and lambda layer build script.